src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -1670,10 +1670,10 @@ size_flags_horizontal = 3
 custom_styles/separator = SubResource( 20 )
 
 [node name="TweakedColourPicker" parent="MicrobeEditorGUI/CellEditor/LeftPanel/MainPanel/VBoxContainer/MarginContainer/EditingPanel/Membrane/ScrollContainer/MarginContainer/VBoxContainer/VBoxContainer2" instance=ExtResource( 14 )]
-margin_left = 24.0
-margin_top = 24.0
-margin_right = 314.0
-margin_bottom = 512.0
+margin_left = 28.0
+margin_top = 28.0
+margin_right = 318.0
+margin_bottom = 516.0
```

Comment:
Accidental change?

Reply:
This is the change that Godot does automatically, I don't know why it keep doing that but it doesn't seem to affect anything thus far. I think we should allow this change from now on.

Reply:
If some margin change consistently happens, I guess that's when we can allow it, but I don't enjoy fixing conflicts if multiple PRs end up with margin changes at the same time.

Reply:
normally, I don't complain either, but this is the only change in this file.
`git checkout master src/microbe_stage/editor/MicrobeEditor.tscn`
`git add src/microbe_stage/editor/MicrobeEditor.tscn `
`git commit -m "reverted MicrobeEditor"`

Reply:
I agree, we should try to avoid it as much as possible but I think we could also spare occasional slips (given there are no other PR with similar change and doesn't cause potential merge conflicts).

Reply:
I've now reverted the change.